flink-core/src/main/java/org/apache/flink/api/java/typeutils/runtime/RowSerializer.java

```diff
@@ -171,6 +181,7 @@ public Row deserialize(DataInputView source) throws IOException {
 				result.setField(i, fieldSerializers[i].deserialize(source));
 			}
 		}
+		result.command = (Command) commandSerializer.deserialize(source);
```

Comment:
should we use the method with reuse variable ?
T deserialize(T reuse, DataInputView source)
though it doesn't perform reusing actually